<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormAddCategory</class>
 <widget class="QWidget" name="FormAddCategory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>103</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>1</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QLineEdit" name="LineEditTitle">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CheckBoxNoValue">
       <property name="text">
        <string>CheckBox</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonAddValue">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>AddValue</string>
       </property>
       <property name="icon">
        <iconset resource="../images/resimg.qrc">
         <normaloff>:/program/program/create.png</normaloff>:/program/program/create.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QScrollArea" name="ScrollAreaValues">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="ScrollAreaLayout">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>498</width>
        <height>99</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QPushButton" name="ButtonCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
       <property name="icon">
        <iconset resource="../images/resimg.qrc">
         <normaloff>:/program/program/cancel.png</normaloff>:/program/program/cancel.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="CheckBoxSelectAll">
       <property name="text">
        <string>SelectAll</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ButtonAccept">
       <property name="text">
        <string>Accept</string>
       </property>
       <property name="icon">
        <iconset resource="../images/resimg.qrc">
         <normaloff>:/program/program/apply.png</normaloff>:/program/program/apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../images/resimg.qrc"/>
 </resources>
 <connections/>
</ui>
